### PR TITLE
Add Coder.com to Full IDE list

### PR DESCRIPTION
**Pull request by Multishifties**
Hey there,

I'd like to request that Coder.com be added to the `Full IDE` list.

Usage is pretty straight forward. Coder also provides video walkthroughs for new users https://www.youtube.com/watch?v=hb_490FC-GU

Essentially each user is given a VM from which to run their code. All computation is handled on the server/VM. This allows everything to happen very quickly. Users can click a button to scale resources on demand. The code editor itself is based on the Monaco editor.

The unique feature that sets this IDE apart would probably be the speed. The ability to scale resources up to the point where a user can compile a linux kernel in under a minute from a Google Chromebook is certainly not something you encounter every day.

Thanks for your time,
-Michael.

(DISCLAIMER : I work as a developer at Coder.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ An ([IDE](https://en.wikipedia.org/wiki/Integrated_development_environment)) is
...
 - [MATLAB Online](https://www.mathworks.com/products/matlab-online.html) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run MATLAB code, visualize results, share scripts and collaborate on mathematical computing software.
 - [Colaboratory](https://research.google.com/colaboratory/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/googlecolab/colabtools) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run Python in a Jupyter notebook with ML libraries and free GPUs for running experiments.
+- [Coder](https://coder.com/) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run most languages on a full Linux VM. Scales your resources (Network, I/O, and up to 96 CPUs) on demand. Syntax highlighting, go-to-definition, live collaboration, and linting are all supported out of the box.
```
Comment (styfle) [inline README.md]: A couple changes to comply with the awesome list guidelines:

1. Remove the forward slash at the end of the URL
2. Change the description so it only has one period at the end.


Here's a good way to revise that:

```
Run most languages on a full Linux VM complete with a terminal, go-to-definition, linting, live collaboration, and auto-scaling.
```

**Comment by styfle** [comment]
Thanks for the PR! 🎉 
coder.com looks nice feels like a full IDE 👌 

I left some comments to revise your entry.

**Comment by Multishifties** [comment]
Hey @styfle Thanks for the feedback! I've made your requested changes. Let me know if there's anything else you'd like changed.

**Comment by styfle** [comment]
Looks good!

It looks like linting is failing because of a change to `awesome-lint` so I will fix in another PR.

**Pull request merged**